prov/rxd/src/rxd_ep.c

```diff
@@ -550,7 +552,6 @@ static ssize_t rxd_ep_post_rts(struct rxd_ep *rxd_ep, struct rxd_x_entry *tx_ent
 	rxd_init_ctrl_pkt(rxd_ep, tx_entry, pkt_entry, RXD_RTS);
 	rts_pkt = rxd_get_ctrl_pkt(pkt_entry);
 	addrlen = RXD_NAME_LENGTH;
-	memset(rts_pkt->ctrl_hdr.source, 0, RXD_NAME_LENGTH);
```

Comment:
I'm assuming you tested these changes with the pingpong and bandwidth benchmarks? I fixed an old bug in the bandwidth test with this memset but I think it was actually related to a separate bug fix that I submitted later.

Reply:
Interesting...., I wonder if the bug was elsewhere as getname should overwrite the string right after.

I didn't test bandwidth I'll do so and report back

Reply:
Interesting, let me test bandwidth, the string should be overwritten right after by fi_getname() call. I'm wondering if there is an error elsewhere.